### PR TITLE
Add log precision

### LINKED ISSUE #1525: [Upgrade to 1.6] JavaScript issues
Since #1524 is fixed, I have now JS and missing files issues:

````
Uncaught TypeError: Cannot read property 'prototype' of undefined
Uncaught TypeError: OpenLayers.Control.Navigation is not a function
GET http://nyon-geoportail-testnew.prod.sig.cloud.camptocamp.net/defs/EPSG21781.js 
GET http://nyon-geoportail-testnew.prod.sig.cloud.camptocamp.net/projCode/somerc.js 
````

I don't know if it's related to my project or the upgrade process since it's the first time I can "see" my project in 1.6...

You can see the errors here http://nyon-geoportail-testnew.prod.sig.cloud.camptocamp.net/

### PR DESCRIPTION
**Pull request by sbrunner**
Fix comment on 8b2bae0604f2a726428ee06dd9b2ddbb5def41e3

### CODE REVIEW

File: c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 1]
```diff
@@ -14,6 +14,7 @@ Version 1.6.2
     WMTSDimension.__acl__ = [DENY_ALL]
 
 2. If you still use version 2 of the print the `.war` file should be updated to the latest version.
+   Gets the `.war` from here: http://pypi.camptocamp.net/print-servlet-2.1-SNAPSHOT-IMG-MAGICK-daeea96d2d3b33d6ccf02fe62d81b9c7aa2574a2.war.
```
Comment (asaunier) [inline c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]: Gets => Get

**Comment by asaunier** [comment]
> Fix #1525

Are you sure? What the connection between a war file and javascript issues?

**Comment by sbrunner** [comment]
Effectively I change it :-)

**Comment by asaunier** [comment]
OK, looks good to me.

**Pull request merged**